buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -122,6 +121,16 @@ RUN wget -O - http://cpanmin.us | perl - --self-upgrade \
   && cpanm Text::CSV \
   && cpanm JSON \
   && cpanm XML::Parser
+
+# Install cmake version 3.11.4
+RUN cd /tmp \
+  && wget https://cmake.org/files/v3.11/cmake-3.11.4.tar.gz \
+  && tar xzf cmake-3.11.4.tar.gz \
+  && rm cmake-3.11.4.tar.gz \
+  && cd /tmp/cmake-3.11.4 \
+  && ./configure \
```

Comment:
Actually, can you investigate turning off the testing that occurs as part of the cmake install?

Reply:
I could not find a way to turn off the testing. @fjeremic Can you merge?